OpenRA.Mods.Common/Widgets/Logic/AssetBrowserLogic.cs

```diff
@@ -212,6 +212,15 @@ public AssetBrowserLogic(Widget widget, Action onExit, World world)
 				prevButton.IsVisible = () => !isVideoLoaded;
 			}
 
+			var node = logic.Value.Nodes.FirstOrDefault(x => x.Key == "SupportedFormats");
+			if (node != null)
+			{
+				var value = node.Value.Value.Replace(" ", "");
+				allowedExtensions = value.Split(',');
+			}
+			else
+				allowedExtensions = new string[0];
```

Comment:
NOTE: It was very late last night. In the end I opted for keeping the default value, so I should have removed the `else`. It is here only because I was going with no default value at one point.